.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
I think it would be nice. It gives a space to start from if people are working in a particular IDE. I use CLion and some of the additional IDE files will define the configurations of how it is to build out the project along with its attachment to the default toolchain etc.